test/artificial_samples/install_cross_compilers_ubuntu_18.04.sh

```diff
@@ -5,4 +5,6 @@ echo "Installing cross compiler for MIPS architecture."
 sudo apt install -y gcc-multilib-mips-linux-gnu g++-7-mips-linux-gnu
```

Comment:
can't we cross compile everything with dockcross? By doing so, we would not need to pollute our systems with the cross compiling toolchains of arm, mips, and so on.

Reply:
if this is correct, then we can remove the other dependencies.

Reply:
Done.